<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>creerDossier</class>
 <widget class="QDialog" name="creerDossier">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>706</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>260</y>
     <width>121</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QTableWidget" name="FormationTable">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>40</y>
     <width>401</width>
     <height>101</height>
    </rect>
   </property>
   <column>
    <property name="text">
     <string>Nom</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Ajouter</string>
    </property>
   </column>
  </widget>
  <widget class="QLabel" name="FormationText">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>10</y>
     <width>351</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Selectionnez les formations auquel vous souhaitez vous inscrire :</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Valider">
   <property name="geometry">
    <rect>
     <x>580</x>
     <y>120</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Valider</string>
   </property>
  </widget>
  <widget class="QTableWidget" name="InscriptionTable">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>40</y>
     <width>401</width>
     <height>101</height>
    </rect>
   </property>
   <column>
    <property name="text">
     <string>UV</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Semestre</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Note</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Ajouter</string>
    </property>
   </column>
  </widget>
  <widget class="QPushButton" name="Suivant">
   <property name="geometry">
    <rect>
     <x>580</x>
     <y>150</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Suivant</string>
   </property>
  </widget>
  <widget class="QLabel" name="InscriptionText">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>10</y>
     <width>151</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Renseignez vos inscriptions :</string>
   </property>
  </widget>
  <widget class="QPushButton" name="AjouterLigne">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>260</y>
     <width>111</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Ajouter une Ligne</string>
   </property>
  </widget>
  <widget class="QTableWidget" name="FormationTable_2">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>150</y>
     <width>401</width>
     <height>101</height>
    </rect>
   </property>
   <column>
    <property name="text">
     <string>Nom</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Supprimer</string>
    </property>
   </column>
  </widget>
  <widget class="QTableWidget" name="InscriptionTable_2">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>150</y>
     <width>401</width>
     <height>101</height>
    </rect>
   </property>
   <column>
    <property name="text">
     <string>UV</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Semestre</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Note</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Supprimer</string>
    </property>
   </column>
  </widget>
  <widget class="QLabel" name="FormationText2">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>70</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Les Formations</string>
   </property>
  </widget>
  <widget class="QLabel" name="Dossier">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>190</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Le Dossier</string>
   </property>
  </widget>
  <widget class="QLabel" name="InscriptionText2">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>70</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Les Inscriptions</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>creerDossier</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>creerDossier</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
